docs/docs/get-started/setup-lightdash/install-lightdash.mdx

```diff
@@ -0,0 +1,173 @@
+---
+sidebar_position: 1
+sidebar_label: Install Lightdash
+---
+import TOCInline from '@theme/TOCInline';
+
+# Install Lightdash
+
```

Comment:
Would be great to add a little summary/table of contents type thing here! Suggested text (but not sure what to write for the one-click deploy bit). 

Also, I _think_ this is what the difference between local vs. production deployment is, but please correct me if I'm wrong! What I'm trying to get across here is like "here's what that means you'd be limited to doing"

```suggestion
There are three ways you can install Lightdash:
1. If you want to run Lightdash in your local machine (that means, you won't be able to share the things you build in Lightdash with anyone else), then you can [get started with our installation script](#deploy-locally-with-our-installation-script).
2. If you want to have Lightdash.... # I don't know what to write here because I'm not sure why I'd choose the 1-click deploy option!
3. If you want to be able to share your Lightdash project with your team, then you can set up a production deployment of Lightdash by following our instructions on [how to deploy Lightdash manually](#deploy-manually).
```

Reply:
@TuringLovesDeathMetal 
Why people would use Heroku deployments ?
- they don't wan't to run Lightdash in their machine so running in Heroku is the simplest way to try Lightdash 
- their company uses Heroku to manage their infrastructure instead of AWS/GCP/etc, so they would run their Lightdash there

Reply:
> What I'm trying to get across here is like "here's what that means you'd be limited to doing"
This might be a bit misleading because the limitations don't depend just on our instructions but where they are using them.
The instructions for the manual deployment can be used in any place that supports docker, being AWS/GCP or their local machine.
E.g  users can deploy Lightdash manually in their local machine, so they won't be able to share what they built.

We can only recommend what instructions to follow when developing and for production but in the end it will depend on their expertise, setup and infrastructure.

Reply:
Okay, that's really helpful information. 

Is it accurate to say:
1. If you want to deploy Lightdash in production, you can use the instructions for the one-click-deploy with Heroku _**or**_ the manually deploy 
2. one-click deploy to Heroku is the simplest way to deploy a Lightdash project in production
3. if you want to deploy a production Lightdash instance on a cloud platform that supports docker and _**isn't**_ Heroku (e.g. AWS or GCP), then you can use our manual deploy instructions 
4. if you want to deploy Lightdash on your local machine, then you should use our install script. But, you can also use our manual deploy instructions if you're really keen.